Makefile

```diff
@@ -4,34 +4,27 @@ flags=-ldflags="-s -w -X main.version=${VERSION}"
 all: build
 
 build:
-	go clean; rm -rf pkg; go build -o udp_server ${flags} udp_server.go
-	go build -o udp_server_monitor ${flags} udp_server_monitor.go
+	go clean; rm -rf pkg; go build -o udp_collector ${flags}
```

Comment:
for all go build commands add `CGO_ENABLED=0 go build ...` in every build action of Makefile. This way your CI/CD will test the build of static executable which you intend to put on k8s.